code/game/area/away_content_areas.dm

```diff
@@ -0,0 +1,127 @@
+
+// Away Missions
+/area/awaymission
```

Comment:
might need to just remove this file with gateway gone

Reply:
we can leave removing unused paths to a second pass, that way it's easier to think of this PR just as "splitting things out" rather than trying to read the diff looking for things that were removed